README.md

```diff
@@ -79,65 +82,77 @@ The Radeon Augmentation Library - [RALI](rali/README.md) is designed to efficien
 MIVisionX provides you with tools for accomplishing your tasks throughout the whole neural net life-cycle, from creating a model to deploying them for your target platforms.
 
 ## Utilities
+
 * [inference_generator](utilities/inference_generator#inference-generator): generate inference library from pre-trained CAFFE models
 * [loom_shell](utilities/loom_shell/README.md#radeon-loomsh): an interpreter to prototype 360 degree video stitching applications using a script
 * [RunVX](utilities/runvx/README.md#amd-runvx): command-line utility to execute OpenVX graph described in GDF text file
 * [RunCL](utilities/runcl/README.md#amd-runcl): command-line utility to build, execute, and debug OpenCL programs
 
 ## Prerequisites
+
 * CPU: [SSE4.1 or above CPU, 64-bit](https://rocm.github.io/hardware.html)
 * GPU: [GFX7 or above](https://rocm.github.io/hardware.html) [optional]
 * APU: Carrizo or above [optional]
 
 **Note:** Some modules in MIVisionX can be built for CPU only. To take advantage of advanced features and modules we recommend using AMD GPUs or AMD APUs.
 
 ### Windows
+
 * Windows 10
 * Windows SDK
 * Visual Studio 2017 and above
 * Install the latest AMD [drivers](https://www.amd.com/en/support)
 * Install [OpenCL SDK](https://github.com/GPUOpen-LibrariesAndSDKs/OCL-SDK/releases/tag/1.0)
 * Install [OpenCV 3.4](https://github.com/opencv/opencv/releases/tag/3.4.0)
-  * Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
-  * Add `%OpenCV_DIR%\x64\vc14\bin` or `%OpenCV_DIR%\x64\vc15\bin` to your `PATH`
+  + Set `OpenCV_DIR` environment variable to `OpenCV/build` folder
+  + Add `%OpenCV_DIR%\x64\vc14\bin` or `%OpenCV_DIR%\x64\vc15\bin` to your `PATH`
 
 ### Linux
+
 * Install [ROCm](https://rocm.github.io/ROCmInstall.html) 
 * ROCm CMake, MIOpenGEMM & MIOpen for Neural Net Extensions (vx_nn)
 * CMake 2.8 or newer [download](http://cmake.org/download/)
 * Qt Creator for [Cloud Inference Client](apps/cloud_inference/client_app/README.md)
 * [Protobuf](https://github.com/google/protobuf) for inference generator & model compiler
-  * install `libprotobuf-dev` and `protobuf-compiler` needed for vx_nn
+  + install `libprotobuf-dev` and `protobuf-compiler` needed for vx_nn
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6035570)